static/js/utils.js

```diff
@@ -1,10 +1,38 @@
 window.onload = _ => {
   const menu = document.querySelector('#menu')
   const avatar = document.querySelector('#avatar-menu')
-  if (menu && avatar ) avatar.addEventListener('click', _ => menu.classList.toggle('active'))
+  if (menu && avatar) avatar.addEventListener('click', _ => menu.classList.toggle('active'))
 }
 
-function notify (type, message, delay) {
+function localize() {
+  let localizeUserPosition
+  const storedPositionJSON = sessionStorage.getItem('currentPosition')
+
+  if (!storedPositionJSON) {
+    navigator.geolocation.getCurrentPosition(async currentPosition => {
+      const res = await fetch('/trips/add/', {
+        method: 'post',
+        body: JSON.stringify([currentPosition.coords.latitude, currentPosition.coords.longitude]),
+        credentials: 'same-origin',
+        headers: { 'Content-Type': 'application/json' }
+      })
+
+      localizeUserPosition = [currentPosition.coords.latitude, currentPosition.coords.longitude]
+      sessionStorage.setItem('currentPosition', JSON.stringify(localizeUserPosition))
+
+      if (window.location.pathname === "{{ url_for_trips(current_user) }}" && res.status === 201)
+        window.location.reload()
+    })
+  }
+  localizeUserPosition = JSON.parse(storedPositionJSON)
+
+  if (window.location.pathname === "{{ url_for('around') }}")
+    moveTo(localizeUserPosition)
+  if (window.location.pathname === "{{ url_for('form_spot') }}")
+    userPosition = localizeUserPosition
+}
```

Comment:
I guess this means that we want a specific action once the usser is localized.
It could be translated with something like:
```
if(userLocalized) userLocalized(localizeUserPosition)
```

Where `userLocalized` would be a function that is specific within the templates "around" and "form_spot"